### ISSUE TITLE
Hermes debugger network tab is always empty

### ISSUE THREAD

**Issue by robertwt7** [facebook/flipper#3105]
## 🐛 Bug Report
Hermes debugger network tab never records any request. is this intentional? 

I can see it using Network plugin, however it is better experience to be able to see it in Hermes debugger that looks like chrome

## To Reproduce
use react native with hermes, open flipper, console is logging, network is not logging

## Environment
Flipper v 0.119.0
Macos 11.6
iOS simulator 15.0
React native 0.65

![Screen Shot 2021-11-29 at 11 23 21 am](https://user-images.githubusercontent.com/15647967/143792503-db8e70e4-4992-4e0e-8402-514018bb963d.png)

**Comment by passy** [comment]
I believe it is. RN uses, by default, platform-specific network stacks like OkHttp for Android. So you will instead see the requests in the Network plugin in Flipper.

**Comment by robertwt7** [comment]
I'm not very familiar with the architecture stuff, but would it be possible to imitate what [RN-debugger](https://github.com/jhen0409/react-native-debugger) is doing? all of the tools including network request is in there but I have to turn off hermes everytime to use it :/ 

@passy

**Comment by NathanNovak** [comment]
Right. I am still trying to figure out how to debug when I set the throttle to Offline.

**Comment by Sir-hennihau** [comment]
any news or fixes on this?
